core/scss/components/sidebar/sidebar.scss

```diff
@@ -459,5 +459,76 @@
         }
       }
     }
+    &--white#{$parent} {
+      background-color: #FCFCFC;
```

Comment:
Let's make this `$gray-10` and then it can be overridden in the product if the designer insists on this shade.